.github/CODEOWNERS

```diff
@@ -1,2 +1,3 @@
 # Global Owners
 * @ansible/devtools
+* @ansible/lightspeed-ansible-vscode
```

Comment:
I hope this does not cause any undesired behaviours. I assumed that the single lint was to be used for root *.